Index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<title> Example Website </title>
+	</head>
+	<body>
+		<h1> Welcome to the Website</h1>
+	</body>
```

Comment:
Please there isn't enough information. if you don't mind make it H2 header and subtitle.